Sources/_OpenAPIGeneratorCore/Translator/RequestBody/translateRequestBody.swift

```diff
@@ -131,7 +139,8 @@ extension TypesFileTranslator {
         requestBody: TypedRequestBody
     ) throws -> Declaration {
         let type = requestBody.typeUsage.typeName
-        let members = try requestBodyContentCases(for: requestBody)
+        let members = try requestBodyContentCases(typeName: type,
+                                                  for: requestBody)
```

Comment:
```suggestion
        let members = try requestBodyContentCases(for: requestBody)
```

As commented above, I don't think we need to pass it in, the type name should already be present on `requestBody`.